tests/e2e/medic-collect.js

```diff
@@ -174,15 +171,13 @@ const rawHttpRequest = rawRequest => {
   });
 };
 
-const saveFormToDb = doc => {
-  return Promise.resolve()
-    .then(() => db.put(doc))
-    .then(res => {
-      const xml = '<xform/>';
-      const body = Buffer.from(xml).toString('base64');
-      return db.putAttachment(doc._id, 'xml', res.rev, body, {
-        type: 'text/xml',
-        length: xml.length,
-      });
-    });
+const saveFormToDb = async doc => {
+  await Promise.resolve();
+  const res = await db.put(doc);
+  const xml = '<xform/>';
+  const body = Buffer.from(xml).toString('base64');
+  return await db.putAttachment(doc._id, 'xml', res.rev, body, {
```

Comment:
With the change to line 174 this function is async and a caller is expecting to await a call to saveFormToDb (this function) and having this return statement return a promise is desirable, rather than this doing the await. As coded here the call to `db.putAttachment` will [return a promise](https://pouchdb.com/api.html#save_attachment) and the `await` will wait for that promise to be resolved to yield a "real value". Since this function is an `await` function though, it will be wrapped in a new already-resolved promise and that is what will be returned. The difference is small, but the preferred approach is only to `await` a call if you need to use the result of it. If not, return that promise and let other code `await` it, which that other code would already be required to do given this is an `async` function.